docs/src/main/asciidoc/_developer-guide/JPA_Reference_Guide.adoc

```diff
@@ -487,35 +481,12 @@ Persistence.xml to use the above custom settings:
 </properties>
 ----
 
-[[replacing-the-current-hibernate-5.x-jars-with-a-newer-version]]
-== Replacing the current Hibernate 5.x jars with a newer version
-
-Just update the current
-wildfly/modules/system/layers/base/org/hibernate/main folder to contain
-the newer version (after stopping your WildFly server instance).
-
-1.  Delete *.index files in
-wildfly/modules/system/layers/base/org/hibernate/main and
-wildfly/modules/system/layers/base/org/hibernate/envers/main folders.
-2.  Backup the current contents of
-wildfly/modules/system/layers/base/org/hibernate in case you make a
-mistake.
-3.  Remove the older jars and copy new Hibernate jars into
-wildfly/modules/system/layers/base/org/hibernate/main +
-wildfly/modules/system/layers/base/org/hibernate/envers/main.
-4.  Update the
-wildfly/modules/system/layers/base/org/hibernate/main/module.xml +
-wildfly/modules/system/layers/base/org/hibernate/envers/main/module.xml
-to name the jars that you copied in.
-5.  Also update the hibernate-infinispan jars in
-wildfly/modules/system/layers/base/org/hibernate/infinispan.
```

Comment:
Should we update this documentation with the new feature packs?

@Sanne ?

Reply:
I wouldn't have too much detail within the Wildfly docs: they get out of date quickly as both ORM and Infinispan release more frequently - and are the ones providing possible "packs" for the existing WildFly.

Might be nice to provide a link to the docs we host at hibernate.org ?
 - https://docs.jboss.org/hibernate/orm/5.3/topical/html_single/wildfly/Wildfly.html

Not sure if Infinispan has a similar one.

Reply:
Thanks, good advice, I remove this detail and link to https://docs.jboss.org/hibernate/orm/5.3/topical/html_single/wildfly/Wildfly.html

Reply:
@scottmarlow better use the /stable/ URL to get the latest version:
https://docs.jboss.org/hibernate/stable/orm/topical/html_single/wildfly/Wildfly.html

Reply:
Excellent, will switch to https://docs.jboss.org/hibernate/stable/orm/topical/html_single/wildfly/Wildfly.html!  Thanks!